test/functional/pdo_sqlsrv/pdo_errorMode.phpt

```diff
@@ -1,39 +1,38 @@
 --TEST--
 Test different error modes. The queries will try to do a select on a table that does not exist on database.
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
+require_once("MsCommon_mid-refactor.inc");
 
-require_once 'MsCommon.inc';
-
-
-function testException(){
+function testException()
+{
     $db = connect();
-    $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
     $sql = "SELECT * FROM temp_table";
-    try{
+    try {
         $q = $db->query($sql);
-    } catch ( Exception $e ){
+    } catch (Exception $e) {
         echo 'Caught exception: ', $e->getMessage();
     }
 }
 
-function testWarning(){
+function testWarning()
+{
     $db = connect();
     $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_WARNING);
     $sql = "SELECT * FROM temp_table";
```

Comment:
this is why I suggest setting ERRMODE in connect() as a parameter, which can take a default value
or simply do NOT set ERRMODE  inside connect()

Reply:
That's what I'm thinking too!! I think default should exception.